largest_sum_contiguous_subarray/LargestSumContiguousSubarray.java

```diff
@@ -0,0 +1,31 @@
+/* 
+   *   Implementation of famous dynamic programming problem
+   *   Largest Sum Contiguous Subarray 
+   *   Time complexity  O(n)
+*/
+
+public class LargestSumContiguousSubarray {
+
+	static int largestSumContiguousSubarray(int[] array) {   //  maximum sum method implemention
+		int prevSum;
+		int currentSum;
+		int i;
+		prevSum = array[0];						// initialize current sum amd previous sum
```

Comment:
max space of the comments would 2 tabs from code.Your comment is inconsistent space.
Better about it [here](http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141999.html#341)